airflow/providers/amazon/aws/hooks/quicksight.py

```diff
@@ -0,0 +1,152 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import time
+from typing import Optional
+
+from botocore.exceptions import ClientError
+
+from airflow import AirflowException
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class QuickSightHook(AwsBaseHook):
+    """
+    Interact with Amazon QuickSight.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHook.
+    .. seealso::
+    :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
+    """
+
+    non_terminal_states = {"INITIALIZED", "QUEUED", "RUNNING"}
+    failed_states = {"FAILED"}
```

Comment:
Constants should be capitalized see: https://www.python.org/dev/peps/pep-0008/#constants

You can also see example for this in EmrHook:
https://github.com/apache/airflow/blob/6126c4e40f36f76d294cfd72f84cb83d97d622a5/airflow/providers/amazon/aws/hooks/emr.py#L106-L116

Reply:
Thanks I agree! I have made constant as caps in other file but missed it over here